cpp/src/groupby/aggregation_operations.hpp

```diff
@@ -19,19 +19,30 @@
 
 #include <limits>
 
-// This header defines the functors that may be used as aggregation operations for 
-// the hash-based groupby implementation. Each functor must define an 'identity value'.
-// The identity value 'I' of an operation 'op' is defined as: for any x, op(x,I) == x.
-// This identity value is used to initialize the hash table values
-// Every functor accepts a 'new_value' which is the new value being inserted into the 
-// hash table and an 'old_value' which is the existing value in the hash table
 
+#ifndef __CUDACC__
```

Comment:
I thought you were going to remove this to avoid it infecting the include chain...  I think an explicit approach like you used in type_dispatcher is safer. We should probably centralize it.

Reply:
Very true, forgot about that. Thanks for catching it.